src/ScriptHeaders/ScriptEnumstwo.h

```diff
@@ -0,0 +1,80 @@
+/******************************************************************************
```

Comment:
Why this filename and not just add these to Enum.h?

Reply:
There is no technical reason to have 2 distinct files, but the original scripting tools from LH have these 2, so keeping the same filenames make it easier to remember from where they came from. If you wish to maintain less files, I could merge them.